plugins/admin/README.adoc

```diff
@@ -69,23 +69,27 @@ Manage Service deployment from a private registry
 For example:
 
 kn admin registry add \
+  --secret-name=[SECRET_NAME]
```

Comment:
How about just `--secret` instead of `--secret-name` ?

Reply:
@navid @12345lcr 
I agree, it's good to keep the flag short. About the wording, we have an issue #53 to track, we can fix it there. Thanks!